### PR TITLE
Update Governance Docs

### PR DESCRIPTION
**Pull request by shemnon**
Add the Hyperledger Code of Conduct, a changelog, and some symbol and
whitespace changes to LICENSE and SECURITY.md.  This allows the repo
linter to pass.

Signed-off-by: Danno Ferrin <danno.ferrin@gmail.com>

### CODE REVIEW

**Comment by grantnoble** [comment]
Do we want to keep the line length less than 100 characters, in keeping with what we are doing with the rest of the docs?

**Comment by shemnon** [comment]
Which specific files are you referring to for the 100 column limit?

The CODE_OF_CONDUCT, LICENSE, and SECURITY files all have specified content, including formatting.

**Comment by grantnoble** [comment]
My bad. I had a very rough look before I went to bed. Tired eyes, should have looked more carefully. 😄

File: SECURITY.md [hunk 1]
```diff
@@ -2,20 +2,19 @@
...
-description of the flaw and any related information (e.g. reproduction
-steps, version) to
+There are two ways to report a security bug. The easiest is to email a description of the flaw and
+any related information (e.g. reproduction steps, version) to
 [security at hyperledger dot org](mailto:security@hyperledger.org).
 
 The other way is to file a confidential security bug in our
-[JIRA bug tracking system](https://jira.hyperledger.org).
-Be sure to set the “Security Level” to “Security issue”.
+[JIRA bug tracking system](https://jira.hyperledger.org). Be sure to set the “Security Level” to
```
Comment (NicolasMassart) [inline SECURITY.md]: We are not using Jira for doc issues anymore

Comment (shemnon) [reply SECURITY.md]: The content of SECURITY.md is mandated, we can't edit it.

Comment (MadelineMurray) [reply SECURITY.md]: @NicolasMassart - Hyperledger Jira will still be used for security issues.

**Pull request merged**